.github/workflows/daily_security.yml

```diff
@@ -2,11 +2,12 @@
 # This file is automatically generated from the templates in stackabletech/operator-templating
 # DON'T MANUALLY EDIT THIS FILE
 # =============
+---
 name: Security audit
 
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '15 4 * * *'
```

Comment:
whats the reason for this? to have it run before the '30 4 * ...' stuff?

Reply:
Not sure, this came from the templating